primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/internal/DefaultWebDriverAdapter.java

```diff
@@ -85,15 +87,21 @@ public WebDriver createWebDriver() {
                 FirefoxOptions firefoxOptions = new FirefoxOptions();
                 firefoxOptions.setPageLoadStrategy(PageLoadStrategy.NORMAL);
                 firefoxOptions.setHeadless(config.isWebdriverHeadless());
-                if (!config.isWebdriverHeadless()) {
-                    firefoxOptions.setCapability(CapabilityType.LOGGING_PREFS, logPrefs);
-                }
+                firefoxOptions.setLogLevel(FirefoxDriverLogLevel.INFO);
```

Comment:
Are we sure we want to modify this?

Reply:
Beside this one thing everything fine and ready to be merged.

Reply:
Yes that is marked as deprecated and a Chrome only option `LoggingPrefs` so I swapped it for the correct Firefox setting which actually still doesn't even work yet.  The Chrome driver is the only one that captures log output right now.